lib/versioning/VersionID.js

```diff
@@ -9,12 +9,12 @@
 // the lengths of the components in bytes
 const LENGTH_TS = 14; // timestamp: epoch in ms
 const LENGTH_SEQ = 6; // position in ms slot
-const LENGTH_SITE = 5; // site identifier
+const LENGTH_TOKEN = 5; // replication group identifier
 
 // empty string template for the variables in a versionId
 const TEMPLATE_TS = new Array(LENGTH_TS + 1).join('0');
 const TEMPLATE_SEQ = new Array(LENGTH_SEQ + 1).join('0');
-const TEMPLATE_SITE = new Array(LENGTH_SITE + 1).join(' ');
+const TEMPLATE_TOKEN = new Array(LENGTH_TOKEN + 1).join(' ');
```

Comment:
NOTE: `TOKEN` here is a bit vague so I changed it to `LENGTH_RG`.

---
It is to be consistent in our use of semantics. 

Although you intended 'SITE' to be a general notion of geography (not necessarily specific to the datacenter), we opted for replication group over site ID to limit confusion because there is a scenario where there may be two nearby deployment sites considered one unit in terms of replication. Is that correct?

If we name the value `replication_group_token` everywhere else and use the same value to build the version ID, it's strange that we call it `SITE_ID` here.

That said, as you're the author of the original code, I'll respect your wishes if you want to retain `SITE_ID` instead.